src/iotjs_binding.c

```diff
@@ -84,6 +84,31 @@ double iotjs_jval_as_number(jerry_value_t jval) {
   return jerry_get_number_value(jval);
 }
 
+bool iotjs_jval_get_data(jerry_value_t jval, iotjs_string_t* getbuffer){
+  if(iotjs_jbuffer_is_bufferwrap(jval)){
+    jerry_value_t to_string = jerry_value_to_string(jval);
```

Comment:
You can get the bytes directly, no need to convert it to string. This is an expensive operations.